src/main/java/com/PhamKornbluhGroup/PublicStashTabBulkAPICaller.java

```diff
@@ -0,0 +1,66 @@
+package com.PhamKornbluhGroup;
+
+import java.io.FileWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Arrays;
+
+public class PublicStashTabBulkAPICaller {
+
+    public APIResultData callAndPrintResults() {
+        //b GGG Get URL
+        HttpURLConnection connection = null;
+        APIResultData resultData = new APIResultData();
+        try {
+            URL url = new URL("https://api.pathofexile.com/public-stash-tabs/pc");
+            connection = (HttpURLConnection) url.openConnection();
+
+            //b Configure connection
```

Comment:
Consider removing comments that don't explain why code behaves a certain way, or comments that don't points out an important nuance.

Reply:
Removed. Thanks